src/Tribe/Tickets_View.php

```diff
@@ -783,12 +783,20 @@ public function get_description_rsvp_ticket( $event_id, $user_id = null, $deprec
 
 		$ticket_count = $this->count_ticket_attendees( $event_id, $user_id );
 
-		if ( ! empty( $rsvp_count ) ) {
-			$descriptions[] = _nx( 'RSVP', 'RSVPs', $rsvp_count, 'Singular and plural texts for RSVP(s)', 'event-tickets' );
+		if ( 1 === $rsvp_count ) {
+			$descriptions[] = tribe_get_rsvp_label_singular( 'tickets_view_description' );
 		}
 
-		if ( ! empty( $ticket_count ) ) {
-			$descriptions[] = _nx( 'Ticket', 'Tickets', $ticket_count, 'Singular and plural texts for Ticket(s)', 'event-tickets' );
+		if ( 1 < $rsvp_count ) {
```

Comment:
```suggestion
		elseif ( 1 < $rsvp_count ) {
```

And can you bump this up to be right after that block above?